cc33conf/suite/trixie/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: cc33conf
+Section: utils
+Priority: optional
+Maintainer: Suhaas Joshi <s-joshi@ti.com>
+Build-Depends: debhelper-compat (= 13), cmake
```

Comment:
Not a CMake project

Reply:
It *wasn't* a CMake project, but it is now :)

I've added CMake support, should make the `rules` file a bit easier too, no more overrides should be needed.

https://git.ti.com/cgit/cc33xx-wlan/cc33xx-utils/commit/?id=34e2cf44e9468bb6308bf163da4cf7349f1717ec